src/features/storycreator/StoryCreator.tsx

```diff
@@ -0,0 +1,50 @@
+import '~/node_modules/react-grid-layout/css/styles.css';
+import '~/node_modules/react-resizable/css/styles.css';
+
+import IntegrationInstructionsOutlinedIcon from '@mui/icons-material/IntegrationInstructionsOutlined';
+import { IconButton } from '@mui/material';
+import { useState } from 'react';
+
+import { useAppDispatch, useAppSelector } from '@/features/common/store';
+import styles from '@/features/storycreator/storycreator.module.css';
+import { selectStoryByID } from '@/features/storycreator/storycreator.slice';
+
+import ButtonRow from './ButtonRow';
+import StoryGUICreator from './StoryGUICreator';
+import StoryTextCreator from './StoryTextCreator';
+
+export default function StoryCreator(props): JSX.Element {
+  const storyID = props.storyID;
+
+  const dispatch = useAppDispatch();
+
+  const story = useAppSelector((state) => {
```

Comment:
story can be undefined when we don't have a matching storyId in store - maybe just redirect to story overview page in that case? - or maybe in `storycreator.page` instead of passing down the story id, pass down the whole story if it exists, and if not, render the overview?